pkg/validation/port_values.go

```diff
@@ -0,0 +1,17 @@
+package validation
+
+import (
+	"fmt"
+
+	"github.com/projectriff/cli/pkg/cli"
+)
+
+func PortValues(port int32, field string) cli.FieldErrors {
+	errs := cli.FieldErrors{}
+
+	if port < 0 || port > 65535 {
```

Comment:
is port 0 valid in k8s?

Reply:
we ignore 0 value - it's treaded as un-specified, since it's an int can't test for ""

Reply:
Validation methods should validate the value passed in. If zero is not a valid value, then it should err. It's up to the caller to understand the difference between a default value and a set value.

Reply:
it's 0 whether they specify it or not, don't know of a way to see what they actually passed in when the flag uses an int value

Reply:
Use an int pointer on the options object if you need to distinguish zero from an unset value.  In this case, the options Validate method is already handling the default case by not calling PortValues.

```suggestion
	if port < 1 || port > 65535 {
```